poplocal/README.txt

```diff
@@ -0,0 +1,15 @@
+About the Poplocal tree
+=======================
+
+The poplocal folder is reserved for locally installed extensions to the
+Poplog system. These extensions are fully integrated with the main system,
+as if they were part of the core system.
+
+The poplocal-tree is not as fully developed as the main Poplog-tree and,
+out of the box, only has two subfolders pre-defined.
```

Comment:
To a new user like myself, I don't know what these statements mean. Does `poplocal` come pre-populated? If so, what does it come with? If not, what does it mean for it to be "not as fully developed as the main Poplog-tree"?

Reply:
Well, what I intended to mean is that the poplocal-tree has fewer pre-defined locations and roles that the main Poplog-tree supplies. Basically it extends fewer search paths. I will try to find a way of expressing that.

Reply:
Done (I hope)